docs/manual/en/introduction/FAQ.html

```diff
@@ -51,5 +51,10 @@ <h2>Why is part of my object invisible?</h2>
 			This could be because of face culling. Faces have an orientation that decides which side is which. And the culling removes the backside in normal circumstances. To see if this is your problem, change the material side to THREE.DoubleSide.
 			<code>material.side = THREE.DoubleSide</code>
 		</p>
+
+		<h2>Why does three.js sometime return strange results for invalid inputs?</h2>
```

Comment:
I think there is a typo: "sometime" instead of "sometimes"

Reply:
I'm not an English native speaker but `sometimes` sounds indeed more correct^^.